subsys/pm/power.c

```diff
@@ -173,8 +173,10 @@ void pm_system_resume(void)
 	 * notification is not required.
 	 */
 	if (!post_ops_done) {
+		int cpu = _current_cpu->id;
+
```

Comment:
Since _current_cpu->id is an uint8_t, it may be preferrable to keep the unsigned property with [cpu] as well.